### DISCUSSION TITLE
What version schema do you use?

### DISCUSSION THREAD

**Discussion by xoxys** [CollaboraOnline/online#4075] [Q&A] [answered]
Hi,

I'm wondering if it is documented somewhere what version schema you use. We use renovate bot to automate our builds and deployments and while the old 6.x version schema was using 4 digits `6.4.14.3` it has now changed to five digits `21.11.0.6.1`. IMO this is pretty confusing especially if it is not really clear what every number means (minor, major, patch, build, ...)

If not documented yet, would you mind explaining a bit the used schema?

**Comment by pedropintosilva** [comment, marked as answer]
Sorry @xoxys I didn't see this earlier, we pay more attention to the topics popping up in https://forum.collaboraonline.com/ I'm answering there

**Reply by pedropintosilva** [reply]
https://forum.collaboraonline.com/t/what-version-schema-do-you-use

**Reply by xoxys** [reply]
Thanks

**Reply by xoxys** [reply]
Sorry to bother you again (and replying here, but I would need to create another account on the forum).

Two days ago, a new version of the docker image was tagged https://hub.docker.com/layers/collabora/code/22.05.0.1/images/sha256-49dd6e729c61390b0e75b9a6a77138a21fc2103a7e232bb7b3e7f5151580a9d5?context=explore where the version schema you have explained doesn't match... `22.05.0.1` has only 4 digits instead of 5 and `22.05` should match `year.month` that doesn't make sense as well. All in all, it makes it pretty hard to automate deployments and version control mechanisms.
